src/main/java/se/sundsvall/esigning/api/model/SigningRequest.java

```diff
@@ -30,8 +33,10 @@ public class SigningRequest {
 	@Schema(description = "Optional descriptive name for the document that is to be signed.", example = "Employment contract")
 	private String name;
 
-	@Schema(description = "The date and time when the signing request expires. Format is yyyy-MM-dd'T'HH:mm:ss.SSSXXX", example = "2024-01-31T12:30:00.000", requiredMode = REQUIRED)
+	@Schema(description = "The date and time when the signing request expires.", example = "2021-12-31T23:59:59Z", requiredMode = REQUIRED)
```

Comment:
If this is not set Comfact will set a default of 30days. Do we need this to be required?

Reply:
Ok, didn't know that. I'll check with Jakob if expiredate should or should not be mandatory.

Reply:
I've spoken to Jakob who has decided that expire attribute should be optional. Adjustments has been made.

Reply:
![image](https://github.com/Sundsvallskommun/pw-e-signing/assets/46994590/caa842eb-8723-4bbb-88e9-c2ee58188629)